pkg/kubefedctl/orphaning/disable.go

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package orphaning
+
+import (
+	"io"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog"
+
+	"sigs.k8s.io/kubefed/pkg/controller/sync"
+	"sigs.k8s.io/kubefed/pkg/kubefedctl/util"
+)
+
+var (
+	orphaning_disable_long = `
+		Removes previously added "orphaning enable" ('kubefed.k8s.io/orphan: true')
+		annotation from a given federated resource.  As result, when the federated 
```

Comment:
s/given//

s/As result, when the federated resource removed,/When the federated resource is subsequently removed/

s/as well//

Reply:
done